gateway/1.4/modules/ROOT/pages/flex-gateway-securing-apis.adoc

```diff
@@ -0,0 +1,20 @@
+= Securing and Governing Flex Gateway APIs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Policies enforce rules when processing requests. This enables you to secure and govern your Flex Gateway APIs.
+
+MuleSoft provides policies that you can apply to your APIs via Anypoint Platform (Connected Mode), or via declarative configuration files (Local Mode). The provided policies include rate limiting, caching, authentication, authorization, threat protection, monitoring, and logging.
+
+For more information about MuleSoft-provided policies, see the xref:policies-included-directory.adoc[].
+
+You can create custom policies based on your specific business requirements and apply them to APIs to extend existing functionality or define new functionality. For information about creating custom policies, see xref:policies-custom-overview.adoc[].
```

Comment:
Consider putting the user goal first:
To extend existing....create custom....

Reply:
Fixed